web/src/components/profile/ProfileAddressLockedFields.tsx

```diff
@@ -0,0 +1,23 @@
+import { AddressContext } from "@/contexts/addressContext";
+import { getMergedConfig } from "@/contexts/configContext";
+import { ReactElement, useContext } from "react";
+
+export const ProfileAddressLockedFields = (): ReactElement => {
+  const { state } = useContext(AddressContext);
+
+  const Config = getMergedConfig();
+  return (
+    <div className="margin-top-4 margin-bottom-4" data-testid={"locked-profileAddressFields"}>
+      <div className="text-bold margin-bottom-05">
+        {Config.profileDefaults.fields.businessAddress.default.header}
+      </div>
+      <div data-testid={"locked-profileAddressLine1"}>{state.addressData.addressLine1}</div>
+      {state.addressData.addressLine2 && (
+        <div data-testid={"locked-profileAddressLine2"}>{state.addressData.addressLine2}</div>
+      )}
+      <div
+        data-testid={"locked-profileAddressMuniStateZip"}
+      >{`${state.addressData.addressMunicipality?.displayName}, ${state.addressData.addressState?.shortCode} ${state.addressData.addressZipCode}`}</div>
+    </div>
+  );
+};
```

Comment:
[boulder]
Given that address is not required during formation, a user who forms and does not provide an address will see `undefined, NJ` in their address field. We should ask product how they would like this represented.

<img width="231" alt="Screenshot 2024-08-05 at 1 21 22 PM" src="https://github.com/user-attachments/assets/c044d6c6-6b06-4b24-b490-59cffe5d40bb">

Reply:
Updated.